docs/src/pages/index.mdx

```diff
@@ -14,7 +14,7 @@ const Comp = React.FC = () => <p>example 1</p>
 
 export default Comp
 
----
+===
```

Comment:
What's this do?

Reply:
separates examples in a single markdown code block.